### PR TITLE
Migrate Merchant Configs Page

### PR DESCRIPTION
**Pull request by AndyClifford**
**Ticket Link:** https://www.notion.so/centrapay/Lanterns-d784e8a5f9fa44328a5d0ae1fd85d37e?p=c6d64de29a2245358a5e9b7eed5edbcc&pm=s

### CODE REVIEW

File: src/content/api/merchant-configs.mdx [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+  </Property>
+</Properties>
+
+---
+
+## Payment Option Config Model
+
+### Attributes
+<Properties>
+  <Property name="type" type="string" required>
```
Comment (MeganSteenkamp) [inline src/content/api/merchant-configs.mdx:54]:
```suggestion
  <Property name="type" type="string">
```

We haven't been adding required attributes on models?

File: src/content/api/merchant-configs.mdx [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+      `walletId` does not belong to a [Settlement Wallet](/api/wallets#settlement-wallets).
+    </Error>
+
+    <Error code="403" message="LIVENESS_MISMATCH">
+      Only [Merchants](https://docs.centrapay.com/api/merchants) with the `test` flag can have merchant configs with test assets, vice versa
+    </Error>
+
+  </Properties>
+
+  <CodePanel slot="code-examples" title="Request" method="POST" path="/api/accounts/{accountId}/members">
```
Comment (MeganSteenkamp) [inline src/content/api/merchant-configs.mdx:124]:
```suggestion
  <CodePanel slot="code-examples" title="Request" method="POST" path="/api/merchants/{merchantId}/configs">
```

File: src/content/api/merchant-configs.mdx [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+  path="/api/merchants/{merchantId}/configs/{configId}"
+>
+  ## Get Merchant Config
+
+  This endpoint allows you to retrieve a Merchant Config by id.
+
+  ### Attributes
+  No attributes.
+
+  ### Errors
```
Comment (MeganSteenkamp) [inline src/content/api/merchant-configs.mdx:189]: @AndyClifford there shouldn't be any errors for this endpoint 👀

File: src/content/api/merchant-configs.mdx [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+  path="/api/merchants/{merchantId}/configs"
+>
+  ## List Merchant Configs
+
+  This endpoint allows you to retrieve a list of Merchant Configs.
+
+  ### Attributes
+  No attributes.
+
+  ### Errors
```
Comment (MeganSteenkamp) [inline src/content/api/merchant-configs.mdx:241]: @AndyClifford there shouldn't be any errors for this endpoint 👀

File: src/content/api/merchant-configs.mdx [hunk 5]
```diff
@@ -0,0 +1,376 @@
...
+    <Error code="403" message="INVALID_WALLET_TYPE">
+      `walletId` does not belong to a [Settlement Wallet](/api/wallets#settlement-wallets).
+    </Error>
+
+    <Error code="403" message="LIVENESS_MISMATCH">
+      Only [Merchants](https://docs.centrapay.com/api/merchants) with the `test` flag can have merchant configs with test assets, vice versa
+    </Error>
+  </Properties>
+
+  <CodePanel slot="code-examples" title="Request" method="PUT" path="/api/merchants/{merchantId}/configs">
```
Comment (MeganSteenkamp) [inline src/content/api/merchant-configs.mdx:326]:
```suggestion
  <CodePanel slot="code-examples" title="Request" method="PUT" path="/api/merchants/{merchantId}/configs/{configId}">
```

File: src/content/api/merchant-configs.mdx (hunk 1 above)
Comment (AndyClifford) [reply src/content/api/merchant-configs.mdx:54]: Yeah I thought this case was slightly different as there is an endpoint that requires an array of this model so it needs to be clear which of the fields are required

**Review by MeganSteenkamp — APPROVED** [review]

**Pull request merged**